docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.4'
+services:
+  gateway:
+    image: 267292272963.dkr.ecr.us-east-1.amazonaws.com/uptycsdev/presto-gateway/presto-gateway:v1.3
```

Comment:
we may want to checkin docker-compose.yml in edna as well as template. instead of putting it out these in public facing org